src/components/DateTimePicker/DateTimePicker.jsx

```diff
@@ -398,7 +409,7 @@ const DateTimePicker = ({
       value.kind = customRangeKind;
     } else {
       const preset = presets
-        .filter(p => p.offset === (clickedPreset ? clickedPreset.offset : selectedPreset))
+        .filter(p => p.id === (clickedPreset ? clickedPreset.id : selectedPreset))
```

Comment:
I think this is a breaking change that affects the public API and thereby the versioning, @davidicus what is your take?

Reply:
yes. I think we can expand this function to check for `id` and if not present use the `offset`.